SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialActorChannel.cpp

```diff
@@ -1283,6 +1282,29 @@ void USpatialActorChannel::ServerProcessOwnershipChange()
 			Channel->ServerProcessOwnershipChange();
 		}
 	}
+
+	const Worker_ComponentId DesiredInterestComponentId = NetDriver->ClassInfoManager->ComputeActorInterestComponentId(Actor);
+
+	auto FindCurrentNCDComponent = [this]()
+	{
+		for (auto ComponentId : NetDriver->ClassInfoManager->SchemaDatabase->NetCullDistanceComponentIds)
```

Comment:
Will make it const 👍 
Think this functionality is nicely contained currently, doesn't expose view to info manager, or vice-versa, so will leave it as is. Can re-evaluate if this functionality is needed elsewhere. Let me know if you disagree.

Reply:
I still think it would be nicer to move this lambda somewhere else as a function in this class, but it's not a particularly strong opinion so happy with this